### PR TITLE
[SPARK-37469][WebUI] unified shuffle read block time to shuffle read fetch wait time in StagePage

### PR DESCRIPTION
**Pull request by toujours33**
### What changes were proposed in this pull request?
Change the metrics name `Shuffle Read Block Time` to `Shuffle Read Fetch Wait Time` in StagePage

### Why are the changes needed?
`ShuffleReadMetrics._fetchWaitTime` shown as "Shuffle Read Block Time" in Stage page, and "fetch wait time" in the SQL page, which make us confused whether shuffle read includes fetch wait & read
Actually `read block time` is just a kind of display name for `fetch wait time`, So we'd better change it in same

### Does this PR introduce _any_ user-facing change?
`shuffleReadBlockTime` metric will be changed into `Shuffle Read Fetch Wait Time` in StagePage

before this patch:
image

after this patch:
image

### How was this patch tested?
Pass GA

### CODE REVIEW

**Comment by sarutak** [comment]
ok to test.

**Comment by sarutak** [comment]
@toujours33 
nit, but could you update the [screenshot](http://spark.apache.org/docs/latest/img/AllStagesPageDetail6.png) in the [doc](spark.apache.org/docs/latest/web-ui.html) ?

**Comment by sarutak** [comment]
Also, could you add the screenshot of before/after this change to the description?

**Comment by toujours33** [comment]
> @toujours33 nit, but could you update the [screenshot](http://spark.apache.org/docs/latest/img/AllStagesPageDetail6.png) in the [doc](spark.apache.org/docs/latest/web-ui.html) ?

ok

File: core/src/main/scala/org/apache/spark/status/storeTypes.scala [hunk 1]
```diff
@@ -129,7 +129,7 @@ private[spark] object TaskIndexNames {
   final val SER_TIME = "rst"
   final val SHUFFLE_LOCAL_BLOCKS = "slbl"
   final val SHUFFLE_READ_RECORDS = "srr"
-  final val SHUFFLE_READ_TIME = "srt"
+  final val SHUFFLE_READ_FETCH_WAIT_TIME = "srfwt"
```
Comment (mridulm) [inline core/src/main/scala/org/apache/spark/status/storeTypes.scala:132]: Wont this not make existing level db files break ?
If yes, we should revert the rename.

Comment (toujours33) [reply core/src/main/scala/org/apache/spark/status/storeTypes.scala:132]: In this scenario, we have not found the problem that the leveldb file cannot be parsed for the time being. But this seems to be a problem, we are glad to revert to this rename

**Comment by toujours33** [comment]
Sorry for the long delay due to some personal reasons, please take a look~
@sarutak @mridulm

**Review by sarutak — APPROVED** [review]

**Comment by sarutak** [comment]
Merged to `master`. Thanks @toujours33 !

**Comment by toujours33** [comment]
Much thanks~

**Pull request closed without merging**